Zend/zend_attributes.c

```diff
@@ -214,6 +214,137 @@ ZEND_API zend_result zend_get_attribute_value(zval *ret, zend_attribute *attr, u
 	return SUCCESS;
 }
 
+static int call_attribute_constructor(
```

Comment:
```suggestion
static zend_result call_attribute_constructor(
```

Reply:
I just copy and pasted the function, but might as well fix this, while I'm at it, thanks.